tests/vocabulary.test.ts

```diff
@@ -7,45 +7,38 @@ describe('Vocabulary tests', function () {
     let client = new VocabularyClient({ endpoint: global.endpointUrl, key: process.env.NEXOSIS_API_TESTKEY });
     this.timeout(5000);
 
-    it('can list vocabularies', () => {
```

Comment:
This one looks like it needs the mochaAsync call in it.

Reply:
Ah.  Done